MODULE.bazel

```diff
@@ -19,6 +19,8 @@ bazel_dep(name = "rules_shell", version = "0.4.1")
 
 bazel_dep(name = "buildifier_prebuilt", version = "8.0.3", dev_dependency = True)
 
+bazel_dep(name = "aspect_bazel_lib", version = "2.19.4")
```

Comment:
I added this to leverage `copy_to_directory` to expose the Kotlin native compiler distribution as a directory in the toolchain https://github.com/bazelbuild/rules_kotlin/pull/1351/files#diff-7c672e82b36304d0c98565a503c039e0ab75d7c924a7961857aea0c9c3666357R36 (as it's massive and has a ton of files)

Reply:
Updated and removed `copy_to_directory`. I've instead used `directory` from bazel-skylib https://github.com/bazelbuild/bazel-skylib/blob/main/rules/directory/directory.bzl which seems to work as well.